ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -155,3 +155,7 @@ def floor(self, name=None):
     )
     def tanh(self, name=None):
         return ivy.tanh(self._ivy_array)
+
+    @with_unsupported_dtypes({"2.4.2 and below": ("float16", "bfloat16")}, "paddle")
+    def all(self, name=None):

```

Comment:
This should have the kwargs axis=None, keepdim=False, name=None (from paddle docs https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/Tensor_en.html)